composer.json

```diff
@@ -16,16 +16,17 @@
     ],
     "require": {
         "php": "^7.1",
-        "php-http/httplug": "^1.0",
+        "php-http/httplug": "^2.0@dev",
         "guzzlehttp/guzzle": "^6.0"
     },
     "require-dev": {
         "ext-curl": "*",
-        "php-http/client-integration-tests": "^0.6"
+        "php-http/client-integration-tests": "dev-master"
```

Comment:
Maybe we should tag a new version from this as well?

Reply:
Yeah